x-pack/plugins/security_solution/server/endpoint/services/actions/clients/lib/base_response_actions_client.ts

```diff
@@ -709,6 +722,53 @@ export abstract class ResponseActionsClientImpl implements ResponseActionsClient
     });
   }
 
+  protected sendActionCreationTelemetry(actionRequest: LogsEndpointAction): void {
+    const isAutomated =
+      (actionRequest.EndpointActions.data.alert_id &&
+        actionRequest.EndpointActions.data.alert_id.length > 0) ??
+      false;
+    this.options.endpointService
+      .getTelemetryService()
+      .reportEvent(ENDPOINT_RESPONSE_ACTION_SENT_EVENT.eventType, {
+        responseActions: {
+          actionId: actionRequest.EndpointActions.action_id,
```

Comment:
Do we really need the action id?

Reply:
Oh wait. Maybe you need it so that we can run reports on which ones were success and wich failed (I see further below the method that handles responses also includes the action id)

Reply:
Yeah, It's good to have the action id to be able to aggregate for responses if needed. The metrics that we need does not really ask for 1:1 correlation between action and response. For v1 we need insights mostly on proportions (how many requests vs how many succeeded/failed) in a given period.